<ui version="4.0" >
 <class>PointWidget</class>
 <widget class="QWidget" name="PointWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>PointWidget</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QCheckBox" name="_stackPoints" >
       <property name="text" >
        <string>Stack Points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="_autoTransform" >
       <property name="text" >
        <string>Auto Transform</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Primitive Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="_primitiveType" >
       <item>
        <property name="text" >
         <string>Point</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Sphere</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Cone</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Disk</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Cube</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Inverted Cone</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Cylinder</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Size Column:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="_sizeColumn" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="_size" >
       <property name="decimals" >
        <number>10</number>
       </property>
       <property name="maximum" >
        <double>1000000000000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Secondary Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="_secondarySize" >
       <property name="decimals" >
        <number>10</number>
       </property>
       <property name="maximum" >
        <double>1000000000000000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Quality</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="_quality" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
